### ISSUE TITLE
CVS parsing stops before end of the file with 1.9.0

### ISSUE THREAD

**Issue by AurelienRibon** [mafintosh/csv-parser#39]
Reproduced easily with a CSV from government:
http://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_month.csv

```js
var csv   = require('csv-parser')
var fs    = require('fs')
var file  = 'all_month.csv'
var count = 0;

fs.createReadStream(file).pipe(csv())
    .on('data', () => count++)
    .on('finish', () => console.log(count));
```

With 1.8.0: 7303 line read (all of them)
With 1.9.0: 5190 line read

This is a huge issue guys. I cannot investigate right now but you should really add big CSV files to your test suite.

**Comment by mafintosh** [comment]
The [test suite is open source](https://github.com/maxogden/csv-spectrum).  Feel free to send a PR that adds a larger csv file

**Comment by AurelienRibon** [comment]
Hey, nice to know there is an acid test for csv parsers, thanks! Indeed they lack big csv files, I'll propose this one.

**Comment by galenandrew** [comment]
Hmm…I'll see if I can debug this later today. @AurelienRibon are you receiving a specific error?

**Comment by Y--** [comment]
@galenandrew nope, there's no error showing up unfortunately

**Comment by galenandrew** [comment]
@mafintosh the issue was related to scope creep with a variable I was using to track quoted cells. PR submitted with a hotfix for this.

**Comment by mafintosh** [comment]
awesome. fixed in 1.9.1

**Comment by vladimirkrasulya-planorama** [comment]
The issue has appeared again in 1.9.2
